zebedee-reader/src/main/java/com/github/onsdigital/zebedee/reader/api/bean/PublishedIndexResponse.java

```diff
@@ -0,0 +1,87 @@
+package com.github.onsdigital.zebedee.reader.api.bean;
+
+import com.github.onsdigital.zebedee.search.indexing.Document;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Object representing a response for the PublishedIndex endpoint
+ */
+public class PublishedIndexResponse {
+
+    private int count;
+    private List<Item> items;
+    private int limit;
+    private int offset;
+    @SerializedName("total_count")
+    private int totalCount;
+
+    public PublishedIndexResponse() {
+        this.items = new ArrayList<>();
+    }
+
+    /**
+     * Adds a list of documents to the response as Items. Also updates the
+     *
+     * @param docs  List of Documents to add to response
+     */
+    public void addDocuments(List<Document> docs) {
+        if (docs == null) {
+            return;
+        }
+        docs.stream()
+                .map(d -> new Item(d.getUri()))
+                .forEach(d -> this.items.add(d));
+
+        count += docs.size();
```

Comment:
shouldn't matter, but would it be slightly more robust just do `count = this.items.size();` here?

Reply:
Yep, that's probably a little simpler 👍